dipy/data/fetcher.py

```diff
@@ -1031,4 +1053,61 @@ def read_cfin_t1():
     """
     files, folder = fetch_cfin_multib()
     img = nib.load(pjoin(folder, 'T1.nii'))
-    return img, gtab
+    return img  # , gtab
+
+
+def read_bundle_atlas_hcp842():
+    """
+    Returns
+    -------
+    file1 : string
+    file2 : string
+    """
+    file1 = pjoin(dipy_home,
+                  'bundle_atlas_hcp842',
+                  'Atlas_in_MNI_Space_16_bundles',
+                  'whole_brain',
+                  'whole_brain_MNI.trk')
+
+    file2 = pjoin(dipy_home,
+                  'bundle_atlas_hcp842',
+                  'Atlas_in_MNI_Space_16_bundles',
+                  'bundles',
+                  '*.trk')
+
+    return file1, file2
```

Comment:
I expect the `read` function to return a Streamlines object and not filename. Indeed, the fetcher already returns file names... I think you should redesign this function.